src/scripts/clipperUI/components/sectionPicker.tsx

```diff
@@ -288,10 +288,9 @@ export class SectionPickerClass extends ComponentBase<SectionPickerState, Sectio
 			}
 		}
 
-		// Display an un-retryable message if API responds with a code that is not retryable
-		// Note that if this is an unhandled api response code, we assume it is not retryable
-		if (this.state.apiResponseCode && !OneNoteApiUtils.isRetryable(this.state.apiResponseCode)) {
-			localizedStrings.notebookLoadFailureMessage = Localization.getLocalizedString("WebClipper.SectionPicker.NotebookLoadUnretryableFailureMessage");
+		// If we can show a better message, especially an actionable one, we do
+		if (this.state.apiResponseCode && OneNoteApiUtils.getLocalizedErrorMessage(this.state.apiResponseCode)) {
```

Comment:
Couldn't we get a `503 Service Unavailable` (API code `10002`) with the GetNotebooks call? That would be retryable (in which I mean, it would help if the user would "try again later"), for example.